### ISSUE TITLE
Result fields

### ISSUE THREAD

**Issue by geminik23** [couchbaselabs/couchbase-rs#51]
How to access the result of LookupInResult?
all fileds are private.

**Comment by daschl** [comment]
you are totally right! This has been an oversight and I'll clean it up. Sorry for that!
